docs/architecture/cloud-native/introduce-eshoponcontainers-reference-app.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: Introducing eShopOnContainers reference app
-description: Architecting Cloud Native .NET Apps for Azure | Introducing eShopOnContainers Reference App
+description: Architecting Cloud-native .NET Apps for Azure | Introducing eShopOnContainers Reference App
```

Comment:
Per previous review comments from Maira, we need to drop the titles from meta description and describe better for SEO.